configure.ac

```diff
@@ -13,6 +13,7 @@ m4_define(exif_minver,                 0.6.14)
 m4_define(exempi_minver,               1.99.5)
 m4_define(gail_minver,                 3.0.0)
 m4_define(notify_minver,               0.7.0)
+m4_define(gtk-layer-shell_minver,      0.8.0)
```

Comment:
I checked and this actually doesn't work as expected.  See https://github.com/cwendling/caja/commit/8e65f5e489fd57c5f9736274d47808ce52131806 for a fix (which is in branch https://github.com/cwendling/caja/commits/wayland-desktop-followup)

Reply:
Switched over to your latest version of configure.ac, will simply apply those two commits as one by hand. Here is the error message yielded by passing --enable-wayland without gtk-layer-shell-dev (in Debian) installed:
```

Package 'gtk-layer-shell-0', required by 'virtual:world', not found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables WAYLAND_CFLAGS
and WAYLAND_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
```
Both wayland and non-wayland builds went fine

Reply:
Cool :+1:
That's exactly what I'm talking about with the error message: it actually tells you how you could fix it if it's not merely a missing dependency, but that pkg-config cannot find it.